src/objectTranslations/Allocation__c-fr.objectTranslation

```diff
@@ -1,73 +1,69 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version='1.0' encoding='UTF-8'?>
 <CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
-    <caseValues>
-        <plural>false</plural>
-        <value>Allocation UCG</value>
-    </caseValues>
-    <caseValues>
-        <plural>true</plural>
-        <value>Allocations UCG</value>
-    </caseValues>
-    <fieldSets>
```

Comment:
@dmkud I noticed that the field set label was not translated on this object. I also noticed that on most objects, the field set label is cryptic and probably not translatable, but that's not the case here.

Reply:
@force2b Allocations is a loan word in French, so it's correct

Reply:
Thank you @dmkud